libs/sdk-ui-kit/src/DatePicker/styles/Datepicker.scss

```diff
@@ -0,0 +1,257 @@
+// (C) 2007-2020 GoodData Corporation
```

Comment:
point to the file name - we should decide whether is should be Datepicker or DatePicker and check the @import in main.scss

Reply:
Renamed as `Datepicker` since is how it was on `goodstrap`